### DISCUSSION TITLE
Why did Github charge me $10 on my first day of using copilot, when the first trial month should be free?

### DISCUSSION THREAD

**Discussion by KjellKod** [community/community#57658] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

### Body

I see that github just charged me $10 on my first day of using Copilot when the first month should be free. Is there a support to reach out to, to dispute this? 

$10 isn't a lot but if they advertise free for one month to get traction, then I expect them to honor this too.

**Comment by KjellKod** [comment, marked as answer]
I'll reach out straight to billing support instead. If other people have the same need, this is the link https://support.github.com/contact/account?tags=hubberfy_billing_and_payments

**Comment by TPerera90** [comment]
I got the same issue today which I'm charged with $10. Just wanted to know that your issue is resolved or not ?

**Comment by HaseebSheikh417** [comment]
same issue

**Comment by kevindiehn** [comment]
same issue

**Comment by minhnguyen-iosdev** [comment]
same issue, they charged me 10$ for the first day trial, wtf

**Comment by fikrirazzaq** [comment]
Seem only for verification, it should be returned a while after
![image](https://github.com/community/community/assets/13093026/bb8e44a0-0717-4155-bd9c-284c662ef987)

**Reply by Cytus2nd** [reply]
what credit card are you using bro, i cant use mine because of GPN.

**Reply by fikrirazzaq** [reply]
> what credit card are you using bro, i cant use mine because of GPN.

I'm using Jago card bro, its Visa Debit card

**Comment by osmfa** [comment]
Same issue

**Comment by lakmal1991** [comment]
same here

**Comment by aishu912** [comment]
Same issue? 
But is it resolved or not?
Or do we have to still pay it for the free trial?

**Comment by lakmal1991** [comment]
I didn’t had repay. And also some said that is just for verification and will return later. Anyhow i cancelled my subscription On Wed, 3 Jul 2024 at 08:49, Aishwarya P Bampalwar ***@***.***> wrote:
Same issue?
But is it resolved or not?
Or do we have to still pay it for the free trial?

**Comment by mojtaba-mohammadi** [comment]
same issue

**Comment by Great-Bird** [comment]
Just got this issue. Under Add-ons at https://github.com/settings/billing/summary, it says my trial is supposed to be free, and my next payment is in a month.

**Comment by akalausichcodes** [comment]
Same issue.

**Comment by diogo0001** [comment]
Same issue

**Comment by xuan-esoft** [comment]
same issue

**Comment by harbakkiran** [comment]
same issue

**Comment by johngr1** [comment]
same issue here.did anyone got an answer about this and what happened?

**Reply by Prem1997123** [reply]
your issue got resolved? I am having same issue.

**Comment by Zero3K** [comment]
This issue is also happening for me. It should only be a $1 charge instead of a $10 one.

**Reply by ReneSambou** [reply]
i also got the same issue. Has yours been resloved?

**Reply by Zero3K** [reply]
No. Not yet.

**Reply by bharath-541** [reply]
any update?

**Reply by MehreenTamboli** [reply]
I got the same issue.

**Comment by patherous** [comment]
same issue

**Comment by nguyenhoaidanh** [comment]
you will get refund after 1-2days

**Comment by xtatsujinx** [comment]
I should not have been charged at all. the payment of $0.00 was made. I didn't have any money in my account to begin with and now i will lose out on the 30 days free because they illegally charged my account.

**Comment by pavelrahman619** [comment]
Same issue

**Reply by leoodaya-filweb** [reply]
Did you get a refund?

**Reply by pavelrahman619** [reply]
> Did you get a refund?

not yet. Will report back if I do

**Reply by Aman-truxie** [reply]
Did you get any reply or refund?

**Reply by pavelrahman619** [reply]
> Did you get any reply or refund?

I got reply. They said they will refund it within 10 days. I am still waiting.

**Reply by leoodaya-filweb** [reply]
Did you guys use credit card or debit card?

**Reply by pavelrahman619** [reply]
I got refunded. 💪💪 

@leoodaya-filweb credit card

**Reply by Aman-mania** [reply]
@pavelrahman619 that's great, how many days it took in total?

**Reply by pavelrahman619** [reply]
@Aman-mania maybe 5-6 days. I only got a hold of my bank statements today

**Reply by rahul-ai-engineer** [reply]
@pavelrahman619 Can you share the steps to get refund? is it auto-refund of we need to raise an request?

**Reply by pavelrahman619** [reply]
> @pavelrahman619 Can you share the steps to get refund? is it auto-refund of we need to raise an request?

It's auto. But since I didn't receive it for a few days I went ahead and raised a ticket to support

**Comment by daruNBC** [comment]
Same here. Never got refunded.

**Comment by aishu912** [comment]
Yep I got the refund

**Reply by PatMiracle** [reply]
How did you go about it?

**Comment by shivix321** [comment]
same issue
